src/component/base/DraftEditor.react.js

```diff
@@ -276,7 +296,7 @@ class DraftEditor extends React.Component {
             onKeyUp={this._onKeyUp}
             onMouseUp={this._onMouseUp}
             onSelect={this._onSelect}
-            ref="editor"
```

Comment:
Let's make sure we grep around for any `this.refs.editor` references in the code - this is scary imo